README.md

```diff
@@ -50,25 +58,25 @@ For more information, please refer to:
 └── template.json
 ```
 
-We use `Typescript` by default for our React applications. Along with the standard files from a `create-react-app` 
-project, the folder structure in the `src` folder is created as per the
-[React Conevention](https://nimblehq.co/compass/development/code-conventions/react/#project-structure).
+`Typescript` is used by default for our React applications.
+To the standard files from a non-ejected `create-react-app` project, this template adds a folder structure in `/src` that follows our [React Convention](https://nimblehq.co/compass/development/code-conventions/javascript/react/#project-structure).
```

Comment:
I just re-read this again and the previous version so I felt that the newer one might not very well converted on the meaning 😅 just with 1 word


```suggestion
With the standard files from a non-ejected `create-react-app` project, this template adds a folder structure in `/src` that follows our [React Convention](https://nimblehq.co/compass/development/code-conventions/javascript/react/#project-structure).
```

Reply:
✅ 9687b2c ;-)